web/wwwroot/js/site.js

```diff
@@ -17,3 +17,12 @@ function checkIfSelectedValueIsTarget(target){
     }
 }
 
+function copyLinkToClipboard(linkToCopy){
+    // Copy the text passed into the function plus the rest of the url
+    let fullLink = window.location.origin+"/invite/"+linkToCopy
+    navigator.clipboard.writeText(fullLink);

```

Comment:
This is actually an asynchronous function and will take a period of time to run, it may be worth putting the 2 copy button changes in a `.then()`, as a result.